packages/components/src/components/table/Table.tsx

```diff
@@ -0,0 +1,169 @@
+import * as React from 'react';
+import { Checkbox, Icon, Dropdown } from '..';
+
+export type TableProps = {
+  items: any[];
+  header: any[];
+  checkbox?: boolean;
```

Comment:
I suggest you to align the naming with "showSorting", "showPagination"
..."showCheckbox" ?